tests/execution/call-graphql-datasource.md

```diff
@@ -0,0 +1,87 @@
+# Call operator with graphQL datasource
+
+#### server:
+
+```graphql
+schema
+  @server(port: 8000, graphiql: true, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true) {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+  user(id: Int!): User
+    @graphQL(baseURL: "http://upstream/graphql", name: "user", args: [{key: "id", value: "{{args.id}}"}])
```

Comment:
```suggestion
  user(id: Int!): User @http(path: "/users/{{args.id}}")
```
Let's use a simpler example here.

Reply:
But in this case, we'd not be testing call with `@graphql`